server/api/src/rsZoneReport.cpp

```diff
@@ -223,8 +223,7 @@ int _rsZoneReport(rsComm_t* _comm, bytesBuf_t** _bbuf)
     zone_arr.push_back(zone_obj);
 
     auto zone = json::object();
-    zone["schema_version"] = (boost::format("%s/%s/zone_bundle.json")
-         % irods::get_server_property<const std::string>("schema_validation_base_uri")
+    zone["schema_version"] = (boost::format("https://irods.org/configuration_schemas/%s/zone_bundle.json")
```

Comment:
Need to agree on which domain to use for all iRODS applications using schemas.
- `https://irods.org`
- `https://schemas.irods.org` _(used by the HTTP API)_
- Or something else

Reply:
Will probably change the schemas to use the same domain as the HTTP API.

Reply:
https://schemas.irods.org only exists to service iRODS 4.2 when it was reaching out to the internet to resolve and validate.  At some point, we were going to decide to retire it?

However, if we choose to point there with new/current plans, then we can plan to keep it alive and resolving.

Reply:
Hmm, why is `schema_version` included at all?

That property isn't part of JSON schema.

Reply:
yeah, can talk that one through together.

Reply:
I'm pretty sure `schema_version` is designed to help clients/tools find the appropriate JSON schema file for validating the zone report.

Are we expecting a client/tool to download the schema based on the provided URL?
Are the schemas going to be hosted somewhere or will they point to the release SHA for irods/irods?

Perhaps we just remove that property from the zone report and document it in the icommand and docs.irods.org?

Developers writing tools can include a copy of the zone report schema file in the packaging or source code of the project to ease usage if they desire.

Schemas are locked for a server release so developers never have to worry about the schema changing unexpectedly. They only need to worry about that whenever dealing with new versions of iRODS. Being forward-compatible can be handled pretty easily on the client-side.

Alternatively, the server could expose an API which allows clients to download the schemas as text. That removes the need for developers to have to think about this entirely because they can ask the server for the info.

With that said, if a zone report is coming from a trusted server, why does the zone report need to be validated? Production systems which are public-facing normally have a CA-backed TLS cert attached to them.

Reply:
After some discussion, we've decided to remove the `schema_version` property.

Created #8038.

Reply:
Done.